app.js

```diff
@@ -2,9 +2,6 @@ const express = require('express');
 const bodyParser = require('body-parser');// initialize our express app
 const app = express();
 
-//set port number
-let port = 1234;
```

Comment:
Does this change port from 1234 to 3000?

Reply:
yes actually the entry point of the app is in /bin/www The server starts in that file with port 3000. Somebody has added this line and it keeps getting merged

Reply:
Hopefully does not brake deployment